db/migrate/20170815232409_clean_up_education_benefits_data_transfer.rb

```diff
@@ -2,16 +2,16 @@ class CleanUpEducationBenefitsDataTransfer < ActiveRecord::Migration[4.2]
   def change
     remove_column(:saved_claims, :education_benefits_claim_id)
 
-    EducationBenefitsClaim.where(saved_claim_id: nil).find_each do |education_benefits_claim|
-      form_type = education_benefits_claim.read_attribute(:form_type)
+    # EducationBenefitsClaim.where(saved_claim_id: nil).find_each do |education_benefits_claim|
+    #   form_type = education_benefits_claim.read_attribute(:form_type)
 
-      education_benefits_claim.saved_claim = SavedClaim::EducationBenefits.form_class(form_type).new(
-        encrypted_form: education_benefits_claim.read_attribute(:encrypted_form),
-        encrypted_form_iv: education_benefits_claim.read_attribute(:encrypted_form_iv)
-      )
+    #   education_benefits_claim.saved_claim = SavedClaim::EducationBenefits.form_class(form_type).new(
+    #     encrypted_form: education_benefits_claim.read_attribute(:encrypted_form),
+    #     encrypted_form_iv: education_benefits_claim.read_attribute(:encrypted_form_iv)
+    #   )
 
-      education_benefits_claim.save!
-    end
+    #   education_benefits_claim.save!
+    # end
```

Comment:
There is a flipper feature flag in this model, and at this point in the migration there *are no tables* yet for flipper, which was added recently. They *shouldnt* be here at all or be a little cleaner and easier to use, but this is pragmatic